Desktop version of portfolio project

miftah1991 opened the pull request:
This is Milestone 4 of the Portfolio project, in this milestone below features have been added.

- Used media queries in `CSS` file to make my desktop version look like template 4 in Figma.
- checked for `linters` and there is no error.
- used grid and flex for layout.
- replicated the Figma design as much as possible.

miftah1991 committed:
Desktop version of portfolio project

julie-ify commented on style.css:
```diff
@@ -347,18 +403,167 @@ textarea {
+  padding-top: 50px;
+}
+
+@media screen and (min-width: 850px) {
+  .proj-1,
```
- [x] Nice one implementing a media query for the desktop breakpoint. However, the required value for `min-width` property should be in range of `992px - 1100px`. This means that you can pick any value between `992px` and `1100px` as the `850px` you have is too small for a desktop breakpoint.

miftah1991 replied:
Thank you I have changed to 992px

julie-ify commented on style.css:
```diff
@@ -3,51 +3,62 @@ body {
+}
+
+/* .mob-version {
   right: 50px;
   cursor: pointer;
   padding-right: 50px;
-}
+} */
```
- [x] Let's make our code clean by removing these commented lines of code.
- [ ] Consider adding padding to all the form input so that it gives it a little space around them.

miftah1991 replied:
I have removed unused code from both files and added  padding to place holders of form input elements

julie-ify commented on style.css:
```diff
@@ -3,51 +3,62 @@ body {
   font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;
 }
 
```
- [x]  Why don't you use the correct font as the `font-family` you have here is quite different from the one in the template. The template has `font-family: Roboto;` and not `font-family: -apple-system, Raleway, Roboto, Oxygen, sans-serif;`. I can see that you have the correct font family in the head of your `HTML` so how you are going to use it in your `CSS` is like this `font-family: 'Roboto', sans-serif;`

miftah1991 replied:
Thank you, really did a big mistake,  I changed my font-family to `font-family: 'Roboto', sans-serif`

julie-ify commented on index.html:
```diff
@@ -10,188 +10,223 @@
-            <h6 class="work-title"><a href="#">Work</a></h6>
-        </div>
-        <h1 class="recent-project-title">MY RECENT WORK</h1>
+        <h2 class="recent-project-title">MY RECENT WORK</h2>
         <hr class="hrbar">
```
- [x] Let's use the correct styling for this element in the desktop version so that your design matches that of your chosen template on the desktop version. Please fix the style using the typography below which I got from your chosen template on Figma.

<img width="104" alt="Loom_zwwbkhcZho" src="https://user-images.githubusercontent.com/81468675/149388232-241e71c0-466b-4f1c-9aff-37a46294326e.png">

miftah1991 replied:
Thanks for your comment, fixed typography for project title as per Figma design

julie-ify commented on index.html:
```diff
@@ -10,188 +10,223 @@
+        
+        <div class="technologies">
+            <ul class="tec-section">
+                <li class="tec lang">
+                    <h2>Langguages</h2>
```
- [x] Let's correct this typo error here from `Langguages` to `Languages` 💯 so that your page looks professional 👍

miftah1991 replied:
Thank you, changed to Languages Languages 👍

julie-ify reviewed (changes requested):
## Hi @miftah1991,

Good job so far!

## Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good Github flow ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

miftah1991 committed:
fix required changes after code review including media min width, remove unused code, padding to all form placeholders, font family, font size to Project title and spelling mistake

miftah1991 commented:
@julie-ify ,

Thank you so much for reviewing my code, appreciate your efforts in reviewing my code, as per your instruction I have made the below changes in my project.
- Media `min-width`
- Removed unused code
- Padding to all form placeholders
- `Font-family`, `font-size`, and `line-height` to Project title 
-  corrected spelling mistake
Regards

mmsesay commented on index.html:
```diff
@@ -10,188 +10,223 @@
-                </div>
-                <div class="proj-btn">
-                    <button type="button" class="btn btn-primary">See Project</button>
+                <div class="proj-det">
+                    <div class="proj-info">
+                        <div class="proj-title">
+                            <h3>Multi-Post Stories</h3>
+                    </div>
+                    <div class="proj-desc">
+                        <p class="project-desc-p">
+                            A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been the industry's standard dummy text ever
+                            since the 1500s, when an unknown printer took a standard dummy text.
+                          </p>
+                    </div>
+                    </div>
+                    <div class="">
+                        <ul class="proj-tech">
+                            <li class="border-class">css</li>
+                            <li class="border-class">html</li>
+                            <li class="border-class">bootstrap</li>
+                            <li class="noborder">ruby</li>
+                        </ul>  
+                    </div>
+                    <div class="proj-btn">
+                        <button type="button" class="btn btn-primary">See Project</button>
+                    </div>
                 </div>
```
- [x] [OPTIONAL] I think it will be nice if you could please give some margin to the projects and increase the size of your project blank card. Your projects section are a bit stretched as compared to the Figma design. 

 Your Design           |
:------------------------:|
![e4](https://user-images.githubusercontent.com/39523312/149410658-7ea7c22a-4f08-4cd5-95e3-2101c1f48a58.png)

 Figma Design           |
:------------------------:|
![e3](https://user-images.githubusercontent.com/39523312/149410670-297ddd46-ff77-4324-a518-51cab35a89a1.png)

miftah1991 replied:
Thank you, I have changed project images size as per Figma Design

mmsesay commented on index.html:
```diff
@@ -10,188 +10,223 @@
-                <div class="proj-btn">
-                    <button type="button" class="btn btn-primary">See Project</button>
-                </div>
+                
```
- [x] [OPTIONAL] I think it will be nice if you could please align your contact form as per the Figma design.

 Your Design           |
:------------------------:|
![e2](https://user-images.githubusercontent.com/39523312/149410550-71a06ed1-3fa3-4496-a9cd-1c383b39ec47.png)

 Figma Design           |
:------------------------:|
![e1](https://user-images.githubusercontent.com/39523312/149410561-eb8a03f1-026f-4fa0-9f55-c76cf14de6bc.png)

miftah1991 replied:
Thank you, Now it is changed.

mmsesay reviewed (approved):
Hi @miftah1991  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

miftah1991 committed:
Add optional changes requested by CR including project images size and contact us message style

miftah1991 commented:
Dear @mmsesay,

Thank you for your kind review, I have considered your optional changes in my project. appreciate your kind efforts in my project code review, your comments help me to make my project more professional.

Regards
